tutorials/nats-rdb-offload/index.mdx

```diff
@@ -0,0 +1,229 @@
+---
+meta:
+  title: Leverage on NATS Messaging Pub/Sub capabilities to offload your Managed Database.
+  description: Learn how to construct a pub/sub architecture that can help you manage the load coming to your database. Dive into this Python tutorial for step-by-step guidance on asynchronous data processing.
+content:
+  h1: Leverage on NATS Messaging Pub/Sub capabilities to offload your Managed Database.
+  paragraph: Learn how to construct a pub/sub architecture that can help you manage the load coming to your database. Dive into this Python tutorial for step-by-step guidance on asynchronous data processing.
+categories:
+  - functions
+  - messaging
+  - postgresql-and-mysql
+tags: terraform scraping-architecture
+dates:
+  validation: 
+  posted: 
+---
+
+## Introduction
+
+In this tutorial, we show how to set up a system that receives a message from a publisher and sends it to a managed database through a buffer subscriber. To do so, we  use Scaleway serverless products and deploy two applications:
+ - A publisher application, that sends a message to a NATS stream created with Scaleway Messaging and Queuing.
+ - A subscriber application, that consumes messages published to the stream and then writes the data into a Scaleway Managed Database.
+
+<Lightbox src="scaleway-scraping-architecture.webp" alt="" />
+
+We show how to provision all the required Scaleway resources using code examples and Scaleway Console [console](https://console.scaleway.com/). The code for the functions is written in Python.
+
+This project exemplifies a decoupled architecture, where producer and consumer perform specific tasks independently. This kind of design is modular and allows for flexibility and scalability. It also adheres to the principles of microservices and serverless architectures, where individual functions or scripts focus on specific tasks.
+
+To properly follow this tutorial, we recommend you to create a dedicated project in your Scaleway account.
+You can name it "RDB and NATS Tutorial". It will hold all your Scaleway ressources created for this tutorial, they will be easier to find.
```

Comment:
```suggestion
You can name it "RDB and NATS Tutorial". It will hold all your Scaleway resources created for this tutorial, so they will be easier to find.
```